src/views/Curators/Curator.vue

```diff
@@ -0,0 +1,537 @@
+<template>
+  <v-container
+    id="curatorPage"
+    fluid
+    class="standard"
+  >
+    <v-row v-if="user().is_curator">
+      <v-col cols12>
+        <v-card v-if="!messages()['getUser'].error">
+          <v-list>
+            <v-list-item class="blue">
+              <v-list-item-content class="pa-0">
+                <v-list-item-title
+                  v-if="user().credentials"
+                  class="headline text-left white--text"
+                >
+                  Welcome, curator {{ user().credentials.username }}
+                </v-list-item-title>
+              </v-list-item-content>
+            </v-list-item>
+          </v-list>
+        </v-card>
+        <v-card>
+          <v-card-text v-if="approvalRequired">
+            <v-card-title
+              id="text-curator-search-0"
+              class="green white--text"
+            >
+              Records awaiting approval
+              <v-spacer />
+              <v-text-field
+                v-model="searches.recordsAwaitingApproval"
+                label="Search"
+                color="white--text"
+                single-line
+                hide-details
+              />
+            </v-card-title>
+            <v-data-table
```

Comment:
In future release, if these tables start getting more complexes, you will probably want to automate the table with a v-for AND//OR create sub component for each table. This doesn't need fixing **yet**.